sdk/transaction_internal.go

```diff
@@ -41,3 +47,1119 @@ func (t *transactionStatusDTOs) toStruct() ([]*TransactionStatus, error) {
 
 	return statuses, nil
 }
+
```

Comment:
@xgreenx @Cempl, do I need to review this file? It seems like DTOs were simply moved without any other changes or am I missing something?

Reply:
Please do not spend a time to review it, just moved files without changes ))